spec/views/posts/posts_index_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe 'posts#index', type: :feature do
+  describe 'Post' do
+    before(:each) do
+      @user1 = User.create(name: 'isaac', photo: 'Tom.png', bio: 'bio', posts_counter: 0, email: 'isaac@gmail.com',
+                           password: 'password')
```

Comment:
- [ ] currently when I run these posts-index tests on this page all of them fail. please make sure that these tests are working to ensure your application runs as expected.